CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## 1.23.0
+* Feature - Add support for ECS Secrets integrating with AWS Secrets Manager [#1713](https://github.com/aws/amazon-ecs-agent/pull/1713)
+* Enhancement - Add availability zone to task metadata endpoint and ECS metadata file
```

Comment:
pr link for this?

Reply:
Added the PR links

Reply:
wait i didn't put pr link because this feature was done by 2 prs

Reply:
ok nvm I saw your update